src/BuildScriptGeneratorCli/Commands/BuildCommand.cs

```diff
@@ -156,6 +158,16 @@ internal override int Execute(IServiceProvider serviceProvider, IConsole console
                 buildInfo.AddDefinition("Repository Commit", sourceRepoCommitId);
             }
 
+            if (!string.IsNullOrWhiteSpace(options.DebianFlavor))
+            {
+                buildInfo.AddDefinition("Os Type", options.DebianFlavor);
+            }
+
+            if (!string.IsNullOrWhiteSpace(options.ImageType))
+            {
+                buildInfo.AddDefinition("Image Type", options.ImageType);
```

Comment:
These will be logged in the build output on every run, similar to repository commit